yt_dlp/extractor/vk.py

```diff
@@ -606,39 +640,109 @@ class VKUserVideosIE(VKBaseIE):
         'url': 'https://vkvideo.ru/playlist/-51890028_-2',
         'only_matching': True,
     }]
-    _VIDEO = collections.namedtuple('Video', ['owner_id', 'id'])
-
-    def _entries(self, page_id, section):
-        video_list_json = self._download_payload('al_video', page_id, {
-            'act': 'load_videos_silent',
-            'offset': 0,
-            'oid': page_id,
-            'section': section,
-        })[0][section]
-        count = video_list_json['count']
-        total = video_list_json['total']
-        video_list = video_list_json['list']
+    API_BASE = 'https://api.vk.com/method'
+    API_Version = 5.269
+
+    def _get_playlist_payload_and_url(self, owner_id, album_id, access_token, offset=0):
+        url = f'{self.API_BASE}/video.getFromAlbum'
+        return url, {
+            'v': self.API_Version,
+            'album_id': album_id,
+            'fields': 'is_esia_verified,is_sber_verified,is_tinkoff_verified,photo_50,verified',
+            'owner_id': owner_id,
+            'access_token': access_token,
+            'offset': offset,
+            'count': 50,
+        }
+
+    def _get_user_payload_and_url(self, url, access_token, start_from=None, section_id=None):
+        if url:
+            return f'{self.API_BASE}/catalog.getVideo', {
+                'v': self.API_Version,
+                'need_blocks': 1,
+                'url': url,
+                'owner_id': 0,
+                'access_token': access_token,
+            }
+        else:
+            return f'{self.API_BASE}/catalog.getSection', {
+                'v': self.API_Version,
+                'need_blocks': 1,
+                'section_id': section_id,
+                'start_from': start_from,
+                'owner_id': 1,
+                'access_token': access_token,
+            }
 
+    def _parse_videos(self, data, video_id):
+        video_data = traverse_obj(data, ('files'), ('video', 'files'))
+        formats, subtitles = self._parse_formats_and_subtiles(video_id, video_data)
+        return {
+            'id': video_id,
+            'formats': formats,
+            'subtitles': subtitles,
+            'title': data.get('title'),
+            'description': data.get('description'),
+            'uploaded_date': data.get('date'),
+            'duration': data.get('duration'),
+            'comments': data.get('comments'),
+            'thumbnails': data.get('image'),
```

Comment:
Is this actually a list of thumbnails? Or:
```suggestion
            'thumbnail': data.get('image'),
```